README.md

```diff
@@ -35,6 +35,7 @@ For a longer list, see:
 For a list of free services and tools for open source .NET projects, see:
 
 * [Free Services & Tools for Open Source .NET Projects](dotnet-free-oss-services.md)
+* [Download the latest Azure .NET SDK](https://www.visualstudio.com/features/azure-tools-vs)
```

Comment:
I don't think the link belongs here, since it's not a "list of free services and tools for open source .NET projects".